core/trino-main/src/test/java/io/trino/sql/planner/optimizations/TestDeterminePartitionCount.java

```diff
@@ -383,7 +385,7 @@ SELECT count(column_a) FROM table_with_stats_a
                 output(
                         node(AggregationNode.class,
                                 exchange(LOCAL,
-                                        exchange(REMOTE, Optional.of(10),
```

Comment:
I think we've lost test coverage here. Is it possible to modify this test so that it still assigns `partitionCount` based on rows count?
I think it's also valuable if we have simple `SELECT` test too (as here)

Reply:
I've changed the aggregation cases to include a `group by` and added simple select / filter / ungrouped count cases in the latest revision.